### DISCUSSION TITLE
Get current Vite dev server instance in global setup

### DISCUSSION THREAD

**Discussion by ahnpnl** [vitest-dev/vitest#2434] [Q&A] [answered]
Hi,

I checked the example of global setup https://github.com/vitest-dev/vitest/tree/main/test/global-setup but there is none of them mentioning about getting the existing Vite dev server instance. How should I do it?

**Comment by sheremet-va** [comment, marked as answer]
You cannot access it

**Reply by ahnpnl** [reply]
thanks!
